src/Connect.js

```diff
@@ -328,18 +341,19 @@ class Connect {
    *   ...
    *  })
    *
-   * @param    {Object}            [verification]           a unsigned verification object, by default the sub is the connected user
-   * @param    {String}            verification.claim       a claim about the subject, single key value or key mapping to object with multiple values (ie { address: {street: ..., zip: ..., country: ...}})
-   * @param    {String}            verification.exp         time at which this verification expires and is no longer valid (seconds since epoch)
-   * @param    {String}            [id='sendVerReq']        string to identify request, later used to get response
+   * @param    {Object}     [verification]        a unsigned verification object, by default the sub is the connected user
+   * @param    {String}     verification.claim    a claim about the subject, single key value or key mapping to object with multiple values (ie { address: {street: ..., zip: ..., country: ...}})
+   * @param    {String}     verification.exp      time at which this verification expires and is no longer valid (seconds since epoch)
```

Comment:
Also is `verification.exp` a `String` or a `Number`?  We claim above that `expiresIn` is a `Number` for `requestDisclosure`, which has me a tad confused.